alicloud/resource_alicloud_ess_scalinggroup.go

```diff
@@ -331,6 +341,9 @@ func buildAlicloudEssScalingGroupArgs(d *schema.ResourceData, meta interface{})
 	if v, ok := d.GetOk("scaling_group_name"); ok && v.(string) != "" {
 		request.ScalingGroupName = v.(string)
 	}
+	if v, ok := d.GetOk("desired_capacity"); ok && v.(int) != 0 {
```

Comment:
it can't be 0?

Reply:
no, it's desired value,the min value is 1.